testing/test_utils.sh

```diff
@@ -46,14 +46,14 @@ function check_setup {
     cat >> $cmd_file <<EOF
 sleep 30
 function test_bacnet {
-    bacnet_base="tcpdump -en -r /scans/test_ping.pcap port 47808"
+    bacnet_base="tcpdump -en -r \$TEST_ROOT/scans/test_ping.pcap port 47808"
```

Comment:
cleaner to use single quotes around entire string rather than escape each $

Reply:
Couldn’t get it working with single quotes. They’re evaluated too early if they’re wrapped in single quotes.  PTAL.